contrib/debian/haos-agent.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Home Assistant OS Agent
+DefaultDependencies=no
+Requires=dbus.socket
```

Comment:
The only difference to the OS service is removed `udisks2.service`.  Maybe we can put that in `Wants=`, but I have to check.

Currently we use udisks2.service for factory reset, which we anyway don't have in Supervised. But in the future we might use it for other things. Could we add it as dependency of this package?

Reply:
true, once udisk lands on the supervisor, but then we should pin that on the service file there?

Reply:
well, we should maybe today add udisk also to the supervisor adr...

Reply:
Added dependency to this package.

Reply:
If/when supervisor access udisks2 directly, then yes we should add it as dependeny in its service file.

I was just wondering, when os-agent starts without udisks, we probably should add it under `Wants=` instead of `Requires=`, and then we probably can use the same service file anyways (as `Wants=` is an optional dependency)

Reply:
I think it is good to have it around already, even for future things like mount management or anything like that.